backend/grant/proposal/models.py

```diff
@@ -13,34 +13,6 @@
 from grant.utils.enums import ProposalStatus, ProposalStage, Category, ContributionStatus
 from grant.settings import PROPOSAL_STAKING_AMOUNT
 
-# Proposal states
-DRAFT = 'DRAFT'
-PENDING = 'PENDING'
-STAKING = 'STAKING'
-APPROVED = 'APPROVED'
-REJECTED = 'REJECTED'
-LIVE = 'LIVE'
-DELETED = 'DELETED'
-STATUSES = [DRAFT, PENDING, STAKING, APPROVED, REJECTED, LIVE, DELETED]
-
-# Funding stages
-FUNDING_REQUIRED = 'FUNDING_REQUIRED'
-COMPLETED = 'COMPLETED'
-PROPOSAL_STAGES = [FUNDING_REQUIRED, COMPLETED]
-
-# Proposal categories
-DAPP = "DAPP"
-DEV_TOOL = "DEV_TOOL"
-CORE_DEV = "CORE_DEV"
-COMMUNITY = "COMMUNITY"
-DOCUMENTATION = "DOCUMENTATION"
-ACCESSIBILITY = "ACCESSIBILITY"
-CATEGORIES = [DAPP, DEV_TOOL, CORE_DEV, COMMUNITY, DOCUMENTATION, ACCESSIBILITY]
-
-# Contribution states
-# PENDING = 'PENDING'
-CONFIRMED = 'CONFIRMED'
```

Comment:
I'm pretty sure these were supposed to be out with the enums... somehow they zombied.